### PR TITLE
Replace postcss-cssnext with postcss-preset-env

### PR DESCRIPTION
**Pull request by Areadrill**

### CODE REVIEW

**Comment by satazor** [comment]
Hello reviewers? Could you take some time to look into this in your spare time? 💪

**Comment by andregoncalvesdev** [comment]
Also, the README should be updated accordingly @Areadrill

File: index.js [hunk 1]
```diff
@@ -27,18 +28,20 @@ module.exports = (options) => {
...
-            // Use CSS next, disabling some features
-            require('postcss-cssnext')({
+            // Use postcss-preset-env, disabling some features
+            require('postcss-preset-env')({
+                stage: 3,
+                insertAfter: {
+                    'nesting-rules': [
+                        // Use `postcss-css-variables` instead of `postcss-custom-properties` because it's more complete
+                        // Note that it must be set after the nesting!
+                        options.cssVariables && require('postcss-css-variables')(options.cssVariables),
```
Comment (andregoncalvesdev) [inline index.js]: Add `postcss-calc` as a dependency and require it here before `postcss-css-variables` and the `calc` with variables will work.

Beware the options too.

**Comment by Areadrill** [comment]
Review plis 🙏

**Comment by satazor** [comment]
Superseded by #28 

Thanks for this PR @Areadrill and sorry for taking too long to review this.

**Pull request closed without merging**